examples/todolist/tests/bindings/test_todolist.kts

```diff
@@ -50,6 +50,29 @@ todo.addItems(listOf("bobo", "fofo"))
 assert(todo.getItems().size == 9)
 assert(todo.getItems()[7] == "bobo")
 
+assert(getDefaultList() == null)
+TodoList().use { todo2 ->
+    setDefaultList(todo)
+    getDefaultList()!!.use { default ->
+        assert(todo.getEntries() == default.getEntries())
+        assert(todo2.getEntries() != default.getEntries())
+    }
+
+    todo2.makeDefault()
+    getDefaultList()!!.use { default ->
+        assert(todo.getEntries() != default.getEntries())
+        assert(todo2.getEntries() == default.getEntries())
+    }
+
+    todo.addItem("Test liveness after being demoted from default")
+    assert(todo.getLast() == "Test liveness after being demoted from default")
+
+    todo2.addItem("Test shared state through local vs default reference")
+    getDefaultList()!!.use { default ->
+        assert(default.getLast() == "Test shared state through local vs default reference")
+    }
+}
+
 // Ensure the kotlin version of deinit doesn't crash, and is idempotent.
 todo.destroy()
 todo.destroy()
```

Comment:
and I guess if you are touching this file you might as well add the trailing \n :)